src/components/v5/common/Table/Table.tsx

```diff
@@ -1,332 +1,23 @@
-import {
-  flexRender,
-  useReactTable,
-  getCoreRowModel as libGetCoreRowModel,
-} from '@tanstack/react-table';
 import clsx from 'clsx';
```

Comment:
We will clarify this as part of #1764 but a reusable table should presumably live in `shared` instead? What do you think?